website/docs/guides/container_service_extension.html.markdown

```diff
@@ -0,0 +1,445 @@
+An example file is provided below, with all the information from the snippets shown in previous steps:
+
+```yaml
+mqtt:
+  verify_ssl: false
+ 
+vcd:
+  host: my-vcd-dev.company.com
+  log: true
+  password: "*****"
+  port: 443
+  username: administrator
+  verify: false
+ 
+service:
+  enforce_authorization: false
+  legacy_mode: false
+  log_wire: false
+  no_vc_communication_mode: true
+  processors: 15
+  telemetry:
+    enable: true
+ 
+broker:
+  catalog: cat-cse
+  ip_allocation_mode: pool
+  network: cse_routed_net
+  org: cse_org
+  remote_template_cookbook_url: https://raw.githubusercontent.com/vmware/container-service-extension-templates/master/template_v2.yaml
+  storage_profile: '*'
+  vdc: cse_vdc
+```
+
+When you execute the following command:
+
+```shell
+cse install -c config.yaml
+```
+
+CSE will use the configuration file from above to install some new custom entities and rights among other required settings.
+You can also refer to the command line [documentation](https://vmware.github.io/container-service-extension) to upload OVA files
+if you skipped the upload with Terraform from previous step.
+
+### Step 6: Rights and roles
+
+You need to publish a new Rights Bundle to your Organization with the new rights that `cse install` command created in VCD.
+The required new rights are listed in the example below. It creates a new bundle with a mix of the existent Default Rights Bundle rights and
+the new ones.
+
+```hcl
+data "vcd_rights_bundle" "default-rb" {
+  name = "Default Rights Bundle"
+}
+
+resource "vcd_rights_bundle" "cse-rb" {
+  name        = "CSE Rights Bundle"
+  description = "Rights bundle to manage CSE"
+  rights = setunion(data.vcd_rights_bundle.default-rb.rights, [
+    "API Tokens: Manage",
+    "Organization vDC Shared Named Disk: Create",
+    "cse:nativeCluster: View",
+    "cse:nativeCluster: Full Access",
+    "cse:nativeCluster: Modify"
+  ])
+  publish_to_all_tenants = true
+}
+```
+
+Once you have the new bundle created, you can now create a specific role for users that will be responsible for managing clusters.
+Notice that the next example is assigning the new rights provided by the new published bundle:
+
+```hcl
+data "vcd_role" "vapp_author" {
+  org  = vcd_org.cse_org.name
+  name = "vApp Author"
+}
+
+resource "vcd_role" "cluster_author" {
+  org         = vcd_org.cse_org.name
+  name        = "Cluster Author"
+  description = "Can read and create clusters"
+  rights = setunion(data.vcd_role.vapp_author.rights, [
+    "API Tokens: Manage",
+    "Organization vDC Shared Named Disk: Create",
+    "Organization vDC Gateway: View",
+    "Organization vDC Gateway: View Load Balancer",
+    "Organization vDC Gateway: Configure Load Balancer",
+    "Organization vDC Gateway: View NAT",
+    "Organization vDC Gateway: Configure NAT",
+    "cse:nativeCluster: View",
+    "cse:nativeCluster: Full Access",
+    "cse:nativeCluster: Modify",
+    "Certificate Library: View" # Implicit role needed
+  ])
```

Comment:
I believe we need this right as well "API Tokens: Manage"

Reply:
You're right, it is in the list 👍